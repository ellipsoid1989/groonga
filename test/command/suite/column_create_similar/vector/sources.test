table_create Terms TABLE_PAT_KEY ShortText \
  --normalizer NormalizerNFKC121 \
  --default_tokenizer TokenNgram

table_create Notes TABLE_NO_KEY
column_create Notes title COLUMN_SCALAR Text

column_create Notes title_terms COLUMN_VECTOR Terms title

table_create NotesSimilar TABLE_NO_KEY
column_create NotesSimilar title COLUMN_SCALAR Text

column_create_similar NotesSimilar title_terms Notes.title_terms

dump
