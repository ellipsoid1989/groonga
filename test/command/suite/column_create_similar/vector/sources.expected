table_create Terms TABLE_PAT_KEY ShortText   --normalizer NormalizerNFKC121   --default_tokenizer TokenNgram
[[0,0.0,0.0],true]
table_create Notes TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Notes title COLUMN_SCALAR Text
[[0,0.0,0.0],true]
column_create Notes title_terms COLUMN_VECTOR Terms title
[[0,0.0,0.0],true]
table_create NotesSimilar TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create NotesSimilar title COLUMN_SCALAR Text
[[0,0.0,0.0],true]
column_create_similar NotesSimilar title_terms Notes.title_terms
[[0,0.0,0.0],true]
dump
table_create Notes TABLE_NO_KEY
column_create Notes title COLUMN_SCALAR Text

table_create NotesSimilar TABLE_NO_KEY
column_create NotesSimilar title COLUMN_SCALAR Text

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenNgram --normalizer NormalizerNFKC121

column_create Notes title_terms COLUMN_VECTOR Terms title
column_create NotesSimilar title_terms COLUMN_VECTOR Terms title
